skietbaan_fe/src/components/ViewGroups.js

```diff
@@ -0,0 +1,163 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import "./groups.css";
+import history from "./history";
+import { passId } from "../actions/postActions";
+import { getname } from "../actions/postActions";
+import { BASE_URL } from "../actions/types";
+class ViewGroups extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      posts: [],
+      newArray: [],
+      count: 0,
+      ShowMe: true,
+      ids: 0,
+      index: 0
+    };
+    this.onBack = this.onBack.bind(this);
+    this.onChange = this.onChange.bind(this);
+    this.delete = this.delete.bind(this);
+    this.editGroup = this.editGroup.bind(this);
+  }
+  UNSAFE_componentWillMount() {
```

Comment:
Spaces between functions, No need for UNSAFE to be all caps

Reply:
this is a predefined function it only works when unsafe is in caps